src/main/java/org/betonquest/betonquest/menu/MenuItem.java

```diff
@@ -116,7 +116,8 @@ protected String of() throws Missing {
             this.descriptions = new HashMap<>();
             try {
                 this.descriptions.putAll(generateDescriptions(menuID.getFullID(), section));
-            } catch (final Missing ignored) {
+            } catch (final Missing e) {
+                LOG.warn("Missing description for menu item " + menuID.getFullID() + " in " + pack.getQuestPath() + "! Reason: " + e.getMessage(), e);
```

Comment:
I think this is slightly confusing, instead say something like this:
```JAVA
LOG.warn("Missing description for menu item '" +  itemID.getFullID() + "' in menu '" + menuID.getFullID() + "' in '" + pack.getQuestPath() + "'! Reason: " + e.getMessage(), e);
```

_I'm not using the suggest feature because I'm not in an IDEA and don't want to suggest invalid code._

Reply:
solved